src/EnergyPlus/main.cc

```diff
@@ -63,6 +63,8 @@ using EnergyPlus::CommandLineInterface::ProcessArgs;
 int
 main( int argc, const char * argv[] )
 {
+	// the following line is only needed when debugging issues related to NaN in Visual Studio. See https://github.com/NREL/EnergyPlus/wiki/Debugging-Tips
+	// unsigned int fp_control_state = _controlfp( _EM_INEXACT, _MCW_EM );
```

Comment:
@JasonGlazer I was thinking of an #ifdef controlled by the build.

@Myoldmopar, comment?